cmd/explaintest/r/explain_easy_stats.result

```diff
@@ -61,15 +61,15 @@ explain delete from t1 where t1.c2 = 1;
 id	count	task	operator info
 Delete_4	N/A	root	N/A
 └─IndexLookUp_11	0.00	root	
-  ├─IndexScan_9	0.00	cop[tikv]	table:t1, index:c2, range:[1,1], keep order:false
-  └─TableScan_10	0.00	cop[tikv]	table:t1, keep order:false
+  ├─IndexScan_9(Build)	0.00	cop[tikv]	table:t1, index:c2, range:[1,1], keep order:false
+  └─TableScan_10(Probe)	0.00	cop[tikv]	table:t1, keep order:false
 explain select count(b.c2) from t1 a, t2 b where a.c1 = b.c2 group by a.c1;
 id	count	task	operator info
 Projection_11	1985.00	root	cast(Column#8)->Column#7
 └─HashLeftJoin_21	1985.00	root	inner join, inner:HashAgg_25, equal:[eq(test.t1.c1, test.t2.c2)]
-  ├─TableReader_34	1999.00	root	data:TableScan_33
+  ├─TableReader_34(Probe)	1999.00	root	data:TableScan_33
```

Comment:
Could we always put the `Build` above the `Probe`?

Reply:
I think the order depends on `left` or `right`?

Reply:
I think if one first executes `explain` as a reference, and then debugs the actual statements step by step, maybe the original format is more clear?

Reply:
it's better to "always put the `Build` above the `Probe`". no matter who is on the left side.

Reply:
Done.